components/todo/AddTodoCard.vue

```diff
@@ -15,10 +15,19 @@
       </div>
 
       <div class="add-todo-card__footer">
-        <button type="submit" class="add-todo-card__footer__btn">
-          {{ $t('add') }}
+        <button type="submit">
+          {{ isTodoEditing ? $t('save') : $t('add') }}
```

Comment:
while this works, using same button for different purpose might prove to be a hassle when we have multiple logic.

Reply:
noted.